persistence/persistence-mybatis-test/src/test/java/org/cbioportal/persistence/mybatis/GeneticProfileMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,196 @@
+                geneticProfile.getDescription());
+        Assert.assertEquals(true, geneticProfile.getShowProfileInAnalysisTab());
+        Assert.assertNull(geneticProfile.getCancerStudy());
+    }
+
+    @Test
+    public void getAllGeneticProfilesDetailedProjection() throws Exception {
+
+        List<GeneticProfile> result = geneticProfileMyBatisRepository.getAllGeneticProfiles("DETAILED", null, null,
+                null, null);
+
+        Assert.assertEquals(6, result.size());
+        GeneticProfile geneticProfile = result.get(0);
+        Assert.assertEquals((Integer) 2, geneticProfile.getGeneticProfileId());
+        Assert.assertEquals("study_tcga_pub_gistic", geneticProfile.getStableId());
+        Assert.assertEquals((Integer) 1, geneticProfile.getCancerStudyId());
+        Assert.assertEquals("study_tcga_pub", geneticProfile.getCancerStudyIdentifier());
+        Assert.assertEquals("COPY_NUMBER_ALTERATION", geneticProfile.getGeneticAlterationType());
+        Assert.assertEquals("DISCRETE", geneticProfile.getDatatype());
+        Assert.assertEquals("Putative copy-number alterations from GISTIC", geneticProfile.getName());
+        Assert.assertEquals("Putative copy-number from GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous " +
+                        "deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.",
+                geneticProfile.getDescription());
+        Assert.assertEquals(true, geneticProfile.getShowProfileInAnalysisTab());
+        CancerStudy cancerStudy = geneticProfile.getCancerStudy();
+        Assert.assertEquals((Integer) 1, cancerStudy.getCancerStudyId());
+        Assert.assertEquals("study_tcga_pub", cancerStudy.getCancerStudyIdentifier());
+        Assert.assertEquals("brca", cancerStudy.getTypeOfCancerId());
+        Assert.assertEquals("Breast Invasive Carcinoma (TCGA, Nature 2012)", cancerStudy.getName());
+        Assert.assertEquals("BRCA (TCGA)", cancerStudy.getShortName());
+        Assert.assertEquals("<a href=\\\"http://cancergenome.nih.gov/\\\">The Cancer Genome Atlas (TCGA)</a> Breast" +
+                " Invasive Carcinoma project. 825 cases.<br><i>Nature 2012.</i> <a href=\\\"http://tcga-data.nci." +
+                "nih.gov/tcga/\\\">Raw data via the TCGA Data Portal</a>.", cancerStudy.getDescription());
+        Assert.assertEquals(true, cancerStudy.getPublicStudy());
+        Assert.assertEquals("23000897", cancerStudy.getPmid());
+        Assert.assertEquals("TCGA, Nature 2012", cancerStudy.getCitation());
+        Assert.assertEquals("SU2C-PI3K;PUBLIC;GDAC", cancerStudy.getGroups());
+        Assert.assertEquals((Integer)0 , cancerStudy.getStatus());
+    }
+
+    @Test
+    public void getAllGeneticProfilesSummaryProjection1PageSize() throws Exception {
+
+        List<GeneticProfile> result = geneticProfileMyBatisRepository.getAllGeneticProfiles("SUMMARY", 1, 0, null, null);
+
```

Comment:
I just prefer leaving a blank line before starting assertions, because they are a different logical group than the data preparation part. And I also like having a blank line after the method definition. Since this method is short, looks like a lot of blank lines, but just tried to be consistent with other methods.